### DISCUSSION TITLE
Report of a few bugs

### DISCUSSION THREAD

**Discussion by Itebayo** [OrcaSlicer/OrcaSlicer#283] [General]
Hello Everybody,

softfiver has some awesome features but I found quite a few bugs the last days.
For example changing the top surface flow ratio, does not change anything at the real print. I have tried this with the same model with very different values. The real piece always looks the same.

Furthermore after finising the outer wall, the printer just travels along the walls a few more times, which is very confusing and costs a lot of print
time. I have added a 3mf file where you can try it out yourself. For example look at the last 250 travel moves at layers 123-127.

[Test Geometrie.zip](https://github.com/SoftFever/BambuStudio-SoftFever/files/10611030/Test.Geometrie.zip)

Are those known issues and will they get fixed?

**Comment by hliebscher** [comment]
have you the same with org BambuStudio?  have you self build the latest version with bug fixes ? or witch version?

i have change the printer to other vendor then it works....

**Reply by Itebayo** [reply]
No the original BambuStudio does not have the wierd travel moves.
And yes I have the latest 1.4.4 bugfix version.
As I was writing and trying out a little more stuff I noticed the issue.

**SETTING "SEAM GAP" to 0% somehow introduces a lot of wierd moves, setting it to 1 % fixes the issue.**
Maybe you can fix this next release :) because at 0% this is quite a lot of extra movement.

For further seam improvement I think a coasting feature would also help quite a bit to reduce the bulge at the end of the movement.

**Comment by julie777** [comment]
@Itebayo 

You should report bugs using issues instead of discussions. They can still have all the same commenting, but they will get noticed faster and certainly get a faster response.
